RIPS/rip-7560.md

```diff
@@ -241,47 +242,54 @@ The amount of gas used by this frame is referred to as `senderValidationGasUsed`
 
 The frame must return 32 bytes `validationData` that is interpreted as:
 
-```solidity
-MAGIC_VALUE_SENDER <20 bytes>, validUntil <6 bytes>, validAfter <6 bytes>
-```
+- **authorizer** (20 bytes) - (magic value/address, defined below)
+- **validUntil** (6 bytes) - a timestamp, or zero for "infinite". The transaction is valid only up to this time.
+- **validAfter** (6 bytes) - a timestamp. The transaction is valid only after this time.
 
-In order to allow a gas estimation of the transaction prior signing, this function
-should avoid reverting on invalid signature, and return `MAGIC_VALUE_SIGFAIL` instead.
+Currently, **authorizer** is treated as a magic value:
+- `MAGIC_VALUE_SENDER` - returned by the account after it verified the transaction (and agrees to pay for its execution)
+- `MAGIC_VALUE_SIGFAIL` - returned by the account if the transaction is syntactically valid, but the signature is not.
+    This valid is used during gas estimation, and it helps wallet and apps to determine the gas used by the validation process.
+- Any other returned value is considered failure, and the transaction is rejected.
+- Future protocol MAY define authorizer as an on-chain contract to perform further account authorization, such as aggregation
 
-Type of the `validUntil` is 6-byte timestamp value, or zero for "infinite". The transaction is valid only up to this time.
-Type of the `validAfter` is 6-byte timestamp. The transaction is valid only after this time.
-
-The `validateTransaction` function can choose to revert on any condition that can be satisfied during gas estimation.
+A revert by `validateTransaction` indicates rejection of the transaction, and it is not included on-chain.
```

Comment:
Revert and invalid return value are equivalent, so they should be in the same element in the list.